Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkDemos/SR-ResearchDemos/SimpleVideo/EyeLink_SimpleVideo.m

```diff
@@ -0,0 +1,396 @@
+function EyeLink_SimpleVideo(screenNumber)
+% Simple video demo with EyeLink integration and animated calibration / drift-check/correction targets.
+% In each trial eye movements are recorded while a video stimulus is presented on the screen.
+% Each trial ends when the space bar is pressed or the video stops playing. A different drift-check/correction
+% animated target is used in each of the 2 trials.
+%
+% Illustrates how a video file can be added for trial play back in Data Viewer's "Trial Play Back Animation" view. 
+%
+% Usage:
+% Eyelink_SimpleVideo(screenNumber)
+% 
+% screenNumber is an optional parameter which can be used to pass a specific value to Screen('OpenWindow', ...)
+% If screenNumber is not specified, or if isempty(screenNumber) then the default:
+% screenNumber = max(Screen('Screens'));
+% will be used.
+
+% Bring the Command Window to the front if it is already open
+if ~IsOctave; commandwindow; end
+
+% Initialize PsychSound for calibration/validation audio feedback
+InitializePsychSound();
+
+% Use default screenNumber if none specified
+if (nargin < 1)
+    screenNumber = [];
+end
+
+% Check if Psychtoolbox is conigured for video presentation:
+AssertOpenGL;

```

Comment:
AssertOpenGL; will be done by the recommended PsychDefaultSetup(), see earlier comments.

Reply:
resolved in 0abf0cbe36691edea3a6d49a87b79149ad7e6f94